localizations/zh-hans.json

```
@@ -0,0 +1,482 @@
+    "Enable quantization in K samplers for sharper and cleaner results. This may change existing seeds. Requires restart to apply.": "在 K 采样器中启用量化以获得更清晰、更清晰的结果。 这可能会改变现有的随机种子。 需要重新启动才能应用",
+    "Emphasis: use (text) to make model pay more attention to text and [text] to make it pay less attention": "强调符：使用 (文字) 使模型更关注该文本，使用 [文字] 使其减少关注",
+    "Use old emphasis implementation. Can be useful to reproduce old seeds.": "使用旧的强调符实现。 可用于复现旧随机种子",
+    "Make K-diffusion samplers produce same images in a batch as when making a single image": "使 K-diffusion 采样器批量生成与生成单个图像时产出相同的图像",
+    "Increase coherency by padding from the last comma within n tokens when using more than 75 tokens": "当使用超过 75 个 token 时，通过从 n 个 token 中的最后一个逗号填空来提高一致性",
+    "Filter NSFW content": "过滤成人内容",
+    "Stop At last layers of CLIP model": "在 CLIP 模型的最后哪一层停下",
+    "Interrogate Options": "逆向生成提示词选项",
+    "Interrogate: keep models in VRAM": "逆向生成: 将模型保存在显存(VRAM)中",
+    "Interrogate: use artists from artists.csv": "逆向生成: 使用 artists.csv 中的艺术家",
+    "Interrogate: include ranks of model tags matches in results (Has no effect on caption-based interrogators).": "逆向生成: 在生成结果中包含与模型标签(tags)相匹配的等级（对基于生成自然语言描述的逆向生成没有影响）",
+    "Interrogate: num_beams for BLIP": "逆向生成: BLIP 的 num_beams",
+    "Interrogate: minimum description length (excluding artists, etc..)": "逆向生成: 最小描述长度（不包括艺术家, 等…）",
+    "Interrogate: maximum description length": "逆向生成: 最大描述长度",
+    "CLIP: maximum number of lines in text file (0 = No limit)": "CLIP: 文本文件中的最大行数（0 = 无限制）",
+    "Interrogate: deepbooru score threshold": "逆向生成: deepbooru 分数阈值",
+    "Interrogate: deepbooru sort alphabetically": "逆向生成: deepbooru 按字母顺序排序",
+    "use spaces for tags in deepbooru": "在 deepbooru 中为标签使用空格",
+    "escape (\\) brackets in deepbooru (so they are used as literal brackets and not for emphasis)": "在 deepbooru 中使用转义 (\\) 括号（因此它们用作文字括号而不是强调符号）",
+    "User interface": "用户界面",
+    "Show progressbar": "显示进度条",
+    "Show image creation progress every N sampling steps. Set 0 to disable.": "每 N 个采样迭代步数显示图像生成进度。 设置 0 禁用",
+    "Show previews of all images generated in a batch as a grid": "以网格的形式预览所有批量生成出来的图片",
+    "Show grid in results for web": "在网页的结果中显示概览图",
+    "Do not show any images in results for web": "不在网页的结果中显示任何图像",
+    "Add model hash to generation information": "将模型的哈希值添加到生成信息",
+    "Add model name to generation information": "将模型名称添加到生成信息",
+    "When reading generation parameters from text into UI (from PNG info or pasted text), do not change the selected model/checkpoint.": "当从文本读取生成参数到 UI（从 PNG 信息或粘贴文本）时，不要更改选定的模型(ckpt)",
+    "Font for image grids that have text": "有文字的概览图使用的字体",
+    "Enable full page image viewer": "启用整页图像查看器",
+    "Show images zoomed in by default in full page image viewer": "在整页图像查看器中默认放大显示图像",
+    "Show generation progress in window title.": "在窗口标题中显示生成进度",
+    "Quicksettings list": "快速设置列表",
+    "Localization (requires restart)": "本地化（需要重新启动）",
```

```suggestion
    "Localization (requires restart)": "本地化（需要重新启动）",
    "ja_JP": "日文",
    "zh-hans": "中文",
```

鉴于日文已合并，可以更好地支持在应用中文本地化后切换语言

---

这个我个人倾向不做任何翻译。保持文件名就可以了。大多数的用户都懂得自己语言的代码简写。